lib/websocket-terminal/.gitignore

```diff
@@ -0,0 +1,6 @@
+websocket-terminal
```

Comment:
I agree that we don't need it. If you tried to skip binary file then problem is that default build `go build` will produce binary with another name